dlp/google/cloud/dlp_v2/proto/dlp.proto

```diff
@@ -280,6 +280,15 @@ service DlpService {
     };
   }
 
+  // Activate a job trigger. Causes the immediate execute of a trigger
+  // instead of waiting on the trigger event to occur.
+  rpc ActivateJobTrigger(ActivateJobTriggerRequest) returns (DlpJob) {
+    option (google.api.http) = {
+      post: "/v2/{name=projects/*/jobTriggers/*}:activate"
+      body: "*"
+    };
+  }
```

Comment:
@crwilcox Why didn't this additional RPC cause an `activate_job_trigger` method to be added in `dlp_service_client.py`?

Reply:
This method is disabled in the GAPIC config, so this is expected behavior. See https://github.com/googleapis/googleapis/blob/fd67be4b51a8c809b97100ccf1c3cffd15b85d7b/google/privacy/dlp/v2/dlp_gapic.yaml#L372-L381 and internal PR 240813399.